apps/src/gamelab/animationListModule.js

```diff
@@ -321,31 +321,19 @@ export function setInitialAnimationList(serializedAnimationList) {
 }
 
 export function addBlankAnimation() {
-  const key = createUuid();
-  return (dispatch, getState) => {
-    // Special behavior here:
-    // By pushing an animation that is "loadedFromSource" but has a null
-    // blob and dataURI, Piskel will know to create a new document with
-    // the given dimensions.
-    dispatch(addAnimationAction(
-      key,
-      {
-        name: generateAnimationName('animation', getState().animationList.propsByKey),
-        sourceUrl: null,
-        frameSize: {x: 100, y: 100},
-        frameCount: 1,
-        looping: true,
-        frameDelay: 4,
-        version: null,
-        loadedFromSource: true,
-        saved: false,
-        blob: null,
-        dataURI: null,
-        hasNewVersionThisSession: false
-      }));
-    dispatch(selectAnimation(key));
-    projectChanged();
-  };
+  return addLibraryAnimation(
+    {
+      name: 'animation',
+      sourceUrl: '/api/v1/animation-library/mUlvnlbeZ5GHYr_Lb4NIuMwPs7kGxHWz/category_backgrounds/blank.png',
```

Comment:
:joy: There's something wonderfully elegant about this.  It also might not be 100% intuitive.  Can we add a comment in `addBlankAnimation()` explaining this strategy of adding a secret, empty library animation?

Reply:
Example library animation props.
```json
{
  "name": "bear",
  "frameCount": 1,
  "frameSize": {
    "x": 254,
    "y": 333
  },
  "looping": true,
  "frameDelay": 2,
  "jsonLastModified": "2017-04-18 05:56:09 UTC",
  "pngLastModified": "2017-04-18 05:56:10 UTC",
  "version": "vCJXAsYgUPWoNTt5h.OKY_tF3cEEY_Sf",
  "sourceUrl": "/api/v1/animation-library/vCJXAsYgUPWoNTt5h.OKY_tF3cEEY_Sf/category_animals/bear.png",
  "sourceSize": {
    "x": 254,
    "y": 333
  }
}
```